### ISSUE TITLE
enrollment challenge password

### ISSUE THREAD

**Issue by morozov-ve** [certnanny/sscep#151]
Hi, how do I add an enrollment challenge password to get a certificate

**Comment by bjanders** [comment]
I don't think this should be asked here, as it is not specific to sscep or an issue with sscep. Something like https://stackoverflow.com/ is better suited for general help requests like this. 

If you are using OpenSSL, then you need to add ``challengePassword`` to your request, for example in the openssl.conf:

```
[ req_attributes ]
challengePassword=4CF4C5D57E1E3B68
```

You can use the sscep ``mkrequest`` script to do this for you, or you can use that as an example for your own script.
